### ISSUE TITLE
[BUG] UI reports "Error loading schema for logging.banzaicloud.io.v1beta1.output.spec.awsElasticsearch.buffer"

### ISSUE THREAD

**Issue by w13915984028** [harvester/harvester#6825]
**Describe the bug**

This issue is observed https://github.com/harvester/harvester/issues/6166#issuecomment-2421374278 by @brandboat 

1. Install Harvester v1.4.0 cluster
2. Enable `rancher-logging` addon
3. Create an output, select any `Output Type`, then click `Output Buffer` page, it always complains about `"Error loading schema for logging.banzaicloud.io.v1beta1.output.spec.awsElasticsearch.buffer"`

![image](https://github.com/user-attachments/assets/e9f887a2-2a7d-496f-affa-c2ee94d27f7b)

![image](https://github.com/user-attachments/assets/aed7972c-0c20-42e1-bf56-3e46f632c3fc)

4. The CRD definition is attached.

**To Reproduce**

**Expected behavior**

**Support bundle**

**Environment**
 - Harvester ISO version:  Harvester v1.4.0
 - Underlying Infrastructure (e.g. Baremetal with Dell PowerEdge R630):

**Additional context**

CRD definition:

```
apiVersion: apiextensions.k8s.io/v1
kind: CustomResourceDefinition
metadata:
  annotations:
    controller-gen.kubebuilder.io/version: v0.6.0
    helm.sh/resource-policy: keep
    meta.helm.sh/release-name: rancher-logging-crd
    meta.helm.sh/release-namespace: cattle-logging-system
    objectset.rio.cattle.io/id: default-mcc-rancher-logging-crd-cattle-fleet-local-system
  creationTimestamp: "2024-10-10T10:32:15Z"
  generation: 1
  labels:
    app.kubernetes.io/managed-by: Helm
    objectset.rio.cattle.io/hash: e5276feeae33cbf06b181e28deb84f85e430a5a8
  name: outputs.logging.banzaicloud.io
  resourceVersion: "3214"
  uid: 9c61c66a-6199-4d6b-a03d-83f518b311f5
spec:
  conversion:
    strategy: None
  group: logging.banzaicloud.io
  names:
    categories:
    - logging-all
    kind: Output
    listKind: OutputList
    plural: outputs
    singular: output
  scope: Namespaced
  versions:
...
    name: v1beta1
    schema: 
      openAPIV3Schema:
        properties:
          apiVersion:
            type: string
          kind:
            type: string
          metadata:
            type: object
          spec:
            properties:
              awsElasticsearch:
                properties:
                  api_key:
                    properties:
                      mountFrom:
                        properties:
                          secretKeyRef:
                            properties:
                              key:
                                type: string
                              name:
                                type: string
                              optional:
                                type: boolean
                            required:
                            - key
                            type: object
                        type: object
                      value:
...
                  application_name:
                    type: string
                  buffer:
                    properties:
                      chunk_full_threshold:
                        type: string
                      chunk_limit_records:
                        type: integer
                      chunk_limit_size:
                        type: string
                      compress:
                        type: string
                      delayed_commit_timeout:
                        type: string
                      disable_chunk_backup:
                        type: boolean
                      disabled:
                        type: boolean
                      flush_at_shutdown:
                        type: boolean
                      flush_interval:
...
```

**Comment by w13915984028** [comment]
@torchiaf @a110605 Please help take a quick look, is this related to UI or backend, thanks.

**Comment by bk201** [comment]
@w13915984028 If the user manually inputs the output buffer, will it work?

**Comment by w13915984028** [comment]
@bk201 Use `kubectl` to add/remove/modify the `clusteroutput/output/clusterflow/flow` is ok (CRDs are there). The UI framework has some changes and forgot to load the related schemas.

**Comment by TachunLin** [comment]
Verified **fixed** on `v1.4-ce5e24e3-head` (24/11/11). Close this issue. 

### Result
<details><summary>$\color{green}{\textsf{PASS}}$ Load ouput buffer content well in Logging -> Outputs $~~$</summary>


  https://github.com/user-attachments/assets/bfaa724c-a9b9-4430-b5f4-5f699a6a0bfe


 </details>

<details><summary>$\color{green}{\textsf{PASS}}$ Load ouput buffer content well in Logging -> Cluster Outputs $~~$</summary>

  https://github.com/user-attachments/assets/a1280323-a6fa-4e91-b0c1-1e0fcd145a56



 </details>

### Test Information
* Test Environment: Single node kvm machine
* Harvester version: `v1.4-ce5e24e3-head` (24/11/11)

### Verify Steps
<details><summary>Load ouput buffer content well in Logging -> Outputs and Cluster Outputs </summary>

  1. Enable rancher-logging addons
  2. Create an Outputs or Cluster Outputs  in Monitoring and Logging -> Logging
  3. Loop through each Output type
  4. Click the Output Buffer
  5. Check the page content 

</details>
